### ISSUE TITLE
Cannot start the JdbcSourceConnector normally in the pipeline

### ISSUE THREAD

**Issue by eechih** [oharastream/ohara#1399]
**Describe the bug**
Cannot start the JdbcSourceConnector normally in the pipeline. Show the following error:
```
Connector configuration is invalid and contains the following 2 error(s): Missing required configuration "source.table.name" which has no default value. source.table.name is required! You can also find the above list of errors at the endpoint `/{connectorType}/config/validate`
```

**To Reproduce**
1. Fill in the following data into the JdbcSourceConnector form
```
jdbc url: jdbc:postgresql://10.2.0.28:5432/postgres
user name: ohara
password: ***
mode: timestamp
timestamp column name: timestamp
Topics: T1
```
2. Click on start button
3. See error

**Expected behavior**

**Screenshots**

**Operation System (please complete the following information):**
 - Backend OS & Version: CentOS 7.2
 - Frontend OS & Version: macOS 10.14.1
 - Browser & Version: Chrome  74.0.3729.169
 - Ohara Version: 0.5-SNAPSHOT

__The fix should go in__
- [x] 0.5.0 (#1407)
- [x] 0.6.0 (#1406)

**Comment by eechih** [comment]
pint @konekoya @wu87988622 Please take a look, thanks.

**Comment by konekoya** [comment]
@eechih I'll fix this as soon as I can. Thanks for raising this issue!

**Comment by konekoya** [comment]
@eechih Closing this as the PR has been merged in 0.5.0! Thanks :)
